samples/bootloader/src/main.c

```diff
@@ -25,6 +25,35 @@
 #endif
 
 
+#ifdef CONFIG_SB_DISABLE_FLASH_PATCH
+static void disable_flash_patch(void)
+{
+	/* Check if register has been written. */
+	if ((NRF_UICR->DEBUGCTRL & UICR_DEBUGCTRL_CPUFPBEN_Msk) != 0x0) {
+		/* Enable flash write. */
+		NRF_NVMC->CONFIG = NVMC_CONFIG_WEN_Wen;
+		__ISB();
+		__DSB();
+
+		/* Write to register. */
+		NRF_UICR->DEBUGCTRL = ~UICR_DEBUGCTRL_CPUFPBEN_Msk;
+
+		/* Wait for flash ready */
+		while (NRF_NVMC->READY == NVMC_READY_READY_Busy)
+			;
+
+		/* Disable flash write. */
+		NRF_NVMC->CONFIG = NVMC_CONFIG_WEN_Ren;
+		__ISB();
+		__DSB();
+
+		/* Reset for change to take effect. */
+		NVIC_SystemReset();
```

Comment:
IS it possible to call the zephyr reset api function?

Reply:
Done